LayoutTests/media/media-src-empty-page-load-hang-expected.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
We would normally add some test here, such as "This test passes if it doesn't get stuck loading permanently."